### PR TITLE
Improve Efficiency

### PR DESCRIPTION
**Pull request by vzts**
First of all, thank you for this wonderful library!
I have some general suggestions on the code. Too many `if`s were kind of bothering.
I've run the tests and it worked well.

Also, could you please release v0.1.0? I think it's ready.
I need `remove_filtered_policy` function that was recently implemented, but it's not on pypi yet...

### CODE REVIEW

File: casbin_sqlalchemy_adapter/adapter.py [hunk 1]
```diff
@@ -20,21 +20,12 @@ class CasbinRule(Base):
...
-            text = text + ', ' + self.v3
-        if self.v4:
-            text = text + ', ' + self.v4
-        if self.v5:
-            text = text + ', ' + self.v5
-        return text
+        arr = [self.ptype]
+        for v in (self.v0, self.v1, self.v2, self.v3, self.v4, self.v5):
+            if not v:
+                break
```
Comment (vzts) [inline casbin_sqlalchemy_adapter/adapter.py]: question: should I just `continue` here?

Comment (vzts) [reply casbin_sqlalchemy_adapter/adapter.py]: hmm let me change this - I think there's possibility of empty string which can break the whole loop.

File: casbin_sqlalchemy_adapter/adapter.py [hunk 2]
```diff
@@ -119,11 +88,12 @@ def remove_filtered_policy(self, sec, ptype, field_index, *field_values):
...
-        if field_index < 0 or len(field_values)>6:
+        if not (0 <= field_index <= 5):
+            return False
+        if not (1 <= field_index + len(field_values) <= 6):
             return False
-        for i in range(len(field_values)):
-            if field_values[i] != "":
-                query = query.filter(getattr(CasbinRule, 'v' + str(field_index + i)) == field_values[i])
+        for i, v in enumerate(field_values):
+            query = query.filter(getattr(CasbinRule, 'v{}'.format(field_index + i)) == v)
```
Comment (techoner) [inline casbin_sqlalchemy_adapter/adapter.py]: Here you need to exclude `v=""`

**Comment by techoner** [comment]
Can you merge the two commits into one?

**Comment by vzts** [comment]
To be conservative and safe, I was trying to make a equivalence to the previous code.
But now I think we should just write the correct code.
I think there's many blind spots for handling empty string field, so I created https://github.com/casbin/casbin/issues/316 in main repo. I will wait for the answer and give a proper update.

**Comment by vzts** [comment]
Hello, could you please check if the latest update is good to go? Thank you.

**Pull request merged**